content/_includes/TCBrowserExceptions.md

```diff
@@ -0,0 +1,26 @@
+---
+---
+
+TrueCommand uses a [self signed certificate](https://tools.ietf.org/html/rfc8705) for a secure connection.
+Because of this, many Internet browsers consider the IP address or DNS host name untrustworthy.
+In these cases, you must add the IP address or DNS host name as an exception to the browser to access the web interface.
+Adding an exception is shown here for two different browsers, but the procedure is similar for most browsers.
```

Comment:
You may want to add/link that this can also be prevent, although not on first login, by adding your own certificate keypair and/or CA.

Reply:
Article edited.